regtest/drr/rt-eabf/config

```diff
@@ -0,0 +1,2 @@
+type=driver
+arg="--plumed plumed.dat --trajectory-stride 1 --timestep 0.005 --ixyz ala12_trajectory.xyz --dump-forces forces --dump-forces-fmt=%10.6f"
```

Comment:
@HanatoK you should add this line to all the `config` files of your regtests:
````
plumed_modules=drr
````

This is to tell plumed that this regtest should be skipped with module drr is not enabled